source/non_matching/coupling.cc

```diff
@@ -40,6 +40,85 @@
 DEAL_II_NAMESPACE_OPEN
 namespace NonMatching
 {
+  namespace internal
+  {
+    /**
+     * Given two triangulations, the first immersed inside the other,
+     * computes and returns the real-space quadrature points of the immersed
+     * triangulation.
```

Comment:
It's not clear to me from the argument list what these two triangulations correlate with. Can you say which of the arguments correspond to which of the two triangulations?

Also, maybe say "Given two ..., THIS FUNCTION computes and returns..."